src/commonMain/kotlin/com/charleskorn/kaml/YamlContextualInput.kt

```diff
@@ -22,17 +22,31 @@ import kotlinx.serialization.descriptors.SerialDescriptor
 import kotlinx.serialization.encoding.CompositeDecoder
 import kotlinx.serialization.modules.SerializersModule
 
-internal class YamlContextualInput(node: YamlNode, yaml: Yaml, context: SerializersModule, configuration: YamlConfiguration) : YamlInput(node, yaml, context, configuration) {
+internal class YamlContextualInput(node: YamlNode, yaml: Yaml, val context: SerializersModule, configuration: YamlConfiguration) : YamlInput(node, yaml, context, configuration) {
```

Comment:
Why does this need to become a property? Can't we get the value of `context` from the `serializersModule` property on `YamlInput`?

```suggestion
internal class YamlContextualInput(node: YamlNode, yaml: Yaml, context: SerializersModule, configuration: YamlConfiguration) : YamlInput(node, yaml, context, configuration) {
```

Reply:
You are right. I'll be working on this.